apps/example/src/App.tsx

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+
+import { Button, Card, Container, PageHeading } from "@tiller-ds/core";
+import { DescriptionList } from "@tiller-ds/data-display";
+import { CheckboxGroup, Input, RadioGroup, Textarea } from "@tiller-ds/form-elements";
+import { Icon } from "@tiller-ds/icons";
+import { Intl } from "@tiller-ds/intl";
+
+function App() {
+  const candidatesHelp = "Lorem ipsum dolor sit amet. Lorem ipsum.";
```

Comment:
I'm not sure if this is necessary. Either have them in translations, or directly as prop without additional const